src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -16,7 +16,13 @@ internal class ListPackageReferencesCommand : CommandBase
         public ListPackageReferencesCommand(
             ParseResult parseResult) : base(parseResult)
         {
-            _fileOrDirectory = parseResult.GetValue(ListCommandParser.SlnOrProjectArgument);
+            _fileOrDirectory = GetAbsolutePath(Directory.GetCurrentDirectory(),
+                parseResult.GetValue(ListCommandParser.SlnOrProjectArgument));
```

Comment:
This means that NuGet side, we require an absolute path. 

We should probably add an absolute path check in the NuGet code that way it's immediately what went wrong when it goes wrong.

Reply:
That is a good idea! Will work on another PR that does that.

Reply:
https://github.com/NuGet/Home/issues/13238 Added a tracking issue